### ISSUE TITLE
Images do not load in some places

### ISSUE THREAD

**Issue by ghost** [davidar/scholarpedia#2]
Here https://github.com/davidar/scholarpedia/wiki/1.-Observational-evidence-for-accretion-discs-in-the-Universe for example there are broken links to images.

Would love to help but don't know how.
Mike

**Comment by davidar** [comment]
Yeah, I didn't upload any of the images to this mirror, you can find them on the original page: http://www.scholarpedia.org/article/Accretion_discs/Observational_evidence_for_accretion_discs_in_the_Universe

**Comment by ghost** [comment]
I uploaded a cuple manually as test to the process and will create a scraper (or clone one with modifications fit for purpose) to upload them. I am working on a local branch and will push when relevant changes are made to the master (yours/thisone).
